### ISSUE TITLE
Trouble with Anchor Jumplink within Fancybox3 iframe in FireFox

### ISSUE THREAD

**Issue by bmiddmjs** [fancyapps/fancyBox#1232]
I cannot get my anchor links to work in a Fanybox3 iframe in Firefox (fine in other browsers).  
Example of page in iframe:
<body>
<a id="page-top"></a>
....content...
<a href="#page-top">Back to Top</a>

 I've tired variations of above (ie <a name= .. and ...page-top" /> and some js fixes I found online but nothing seems to work. I've read that this is an issue in Firefox but I'm upgrading from Fancybox1 and the links work fine there so I thought I'd see if there were any suggestions.

**Comment by bmiddmjs** [comment]
sorry...should have previewed it..
example:
< body>
< a id="page-top">< /a>
...content...
< a href="#page-top">Back to Top< /a>
< /body>

**Comment by fancyapps** [comment]
I always wonder what people are thinking while creating issues like this. Do you really think that the script is doing some magic and is creating images/iframed pages in some mysterious ways? But it just creates a markup and the rest is up to the browser!

Anyway, if you would just spend a little time googling, you would have stumbled upon this this [bugzilla entry](https://bugzilla.mozilla.org/show_bug.cgi?id=638598). This [stackoverflow answer](http://stackoverflow.com/questions/9687085/jump-link-inside-an-iframe) has some interesting info, too.

In conclusion, I suggest you to use JS `scrollTo()` to overcome this Firefox "feature".

**Comment by fancyapps** [comment]
btw, I was thinking about this and found out that anchor links start working if iframe has scrollbars (instead of the parent element). So, if you enable iframe scrolling and set height to fit inside viewport, like this:  
```
iframe : {
    scrolling : 'yes',
    css : {
        height: '80vh'
    }
}
```

then it starts working!

**Comment by bmiddmjs** [comment]
Thanks... I did spend a 1/2 day trying various solutions from the internet (including several js) before I decided to post. It may be a testament to my poor programming skills but I thought that since it worked in fancybox1 someone would have the same problem and a workaround for version 3 since I could not find one that I could get to work.  Maybe your point is that it is not an issue with fancybox3 but with programming implementation and therefore should not be posted as an "issue". Also, as you suggested above, my fancybox1 did use scrollbars in the iframe. I do appreciate your time and input.

**Comment by fancyapps** [comment]
Sorry, if I sounded too harsh. It is just that my initial thoughts were "oh, here we go again, another one asking me to fix a browser or change web standards". 
I hope it is now working fine for you.

**Comment by bmiddmjs** [comment]
You mean you can't do that??  Seriously, I understand and it is indeed working for me now. I'm a big boy and can handle some criticism :) Thanks again.
